Exec/radiation_tests/RadBreakout/Prob_nd.F90

```diff
@@ -13,43 +13,11 @@ subroutine amrex_probinit(init, name, namlen, problo, probhi) bind(C, name="amre
   integer untin, i
```

Comment:
untin can be removed

Reply:
untin is also used for opening up the model file, so I kept it.